modules/social_features/social_event/modules/social_event_addtocal/src/Controller/AddToCalendarIcsController.php

```diff
@@ -63,10 +63,10 @@ public function downloadIcs() {
     $dates = $this->request->get('dates');
     $timezone = $this->request->get('timezone');
 
-    // Generate DTSTAMP, which needs to be the time the ICS object created.
+    // Generate timestamp, which needs to be the time the ICS object created.
     // https://icalendar.org/iCalendar-RFC-5545/3-6-1-event-component.html
     $now = new \DateTime('now', new \DateTimezone('UTC'));
-    $dtstamp = $now->format('Ymd\THis\Z');
+    $timestamp = $now->format('Ymd\THis\Z');
```

Comment:
We could also use `$dateTimeStamp` which CSpell should accept as well, but I think in this case timestamp is fine (even though we usually use that for UNIX timestamps and not YMDHis).